content/en/docs/setup/install/multicluster/single-network/initial-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Initial Configuration
+description: Describes how to complete the initial configuration for a multicluster mesh on a flat network.
+weight: 10
+keywords: [multicluster,first-cluster]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, configure a
+single cluster {{< gloss >}}service mesh{{< /gloss >}} as your starting point.
+
+The single cluster mesh you deploy in this section, serves as a model for the
+configuration of the first {{< gloss >}}primary cluster{{< /gloss >}}. You need
```

Comment:
Why doesn't it just serve as the first primary cluster?

Reply:
+1 that's exactly what this should be. I think that we should probably rephrase things to make sure this is more clear. We can probably drop the concept of "single cluster service mesh" altogether. Maybe we can replace the first 3 paragraphs with something like:

```
To build a service mesh, we begin by installing our first primary cluster.
```

I purposely left "multicluster" out of this, since this page will eventually be the starting point for "install", not specifically "multicluster installation".

Reply:
This was feedback I got from the TOC. This shouldn't be anyone's "first primary cluster" since folks reading this document must be familiar with single cluster meshes. Those already have a primary cluster, and members of the TOC found it confusing to start this guide with a "first" cluster of any kind. I explicitly removed all mentions of installing Istio from the introduction of this page because of feedback from the TOC that the content should be accessible for folks with an existing single cluster mesh.

Calling this doc the initial configuration is accurate since it is the set of steps folks need to complete to configure a cluster to start their multicluster deployment. 

Currently, people starting out should go to the getting started and deploy a single cluster mesh. Multicluster is not for them at the moment.

@craigbox @nmittler The potential future transformation of this page into something more suitable for beginners should be handled separately at a later date.